spec/support/shared_examples.rb

```diff
@@ -25,4 +25,21 @@
   it 'generates a token upon creation' do
     expect(object.token).to be_present
   end
+end
+
+shared_examples 'doesnt_create_new_user' do
+  it 'does not create new User record' do
+    signup_handle.signup('fake_sripe_token')
+    expect(User.all.count).to eq(0)
+  end
+
+  it 'does not send welcome email' do
+    signup_handle.signup('fake_sripe_token')
+    expect(ActionMailer::Base.deliveries).to be_empty
+  end
+
+  it 'sets @status to :failed' do
+    signup_handle.signup('fake_sripe_token')

```

Comment:
Your typo exists all the way down here, which leads me to believe that autocomplete bit you, not copy and paste.